common.mk

```diff
@@ -113,7 +116,7 @@ optee-os-common:
 
 OPTEE_OS_CLEAN_COMMON_FLAGS ?= $(OPTEE_OS_COMMON_FLAGS)
 
-optee-os-clean-common:
+optee-os-clean-common: xtest-clean
```

Comment:
When we clean everything, we will clean both `optee_os` and `optee_test`. But the makefile of optee_test depends on the devkit in optee_os. So if optee_os is cleaned first, xtest-clean will fail.